### PR TITLE
Add tests for `One` and `Zero`

### PR DESCRIPTION
**Pull request by lkdvos**

### CODE REVIEW

**Comment by lkdvos** [comment]
Can you still check for invalidations?

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         arch:
           - x64
     steps:
-      - uses: actions/checkout@v2
```
Comment (Jutho) [inline .github/workflows/CI.yml:29]: I've seen somewhere that there is also a bot that automatically makes a PR to update these GitHub actions. Not sure if it can also update itself :-).

**Comment by Jutho** [comment]
I will check for invalidations after merge; it anyway needs another format run as well.
